src/plugins/legacy/reformat/medCropToolBox.cpp

```diff
@@ -111,7 +112,12 @@ medCropToolBox::medCropToolBox(QWidget* parent)
     d->borderWidget = vtkSmartPointer<vtkBorderWidget>::New();
     d->borderWidget->CreateDefaultRepresentation();
     d->borderWidget->SelectableOff();
-    static_cast<vtkBorderRepresentation*>(d->borderWidget->GetRepresentation())->GetBorderProperty()->SetColor(0,1,0);
+
+    vtkNew<vtkNamedColors> colors;
+    vtkNew<vtkBorderRepresentation> rep;
```

Comment:
If you're manually creating the border widget then the call to `CreateDefaultRepresentation` should be removed.

Reply:
Done